charts/gardener/gardenlet/charts/runtime/templates/deployment.yaml

```
@@ -21,6 +21,10 @@ spec:
     metadata:
       annotations:
         checksum/configmap-gardenlet-config: {{ include (print $.Template.BasePath "/configmap-componentconfig.yaml") . | sha256sum }}
+        {{- if .Values.global.gardenlet.metrics.enabled }}
+        prometheus.io/scrape: 'true'
+        prometheus.io/port: {{ .Values.global.gardenlet.metrics.port }}
```

Should this be quoted?

```suggestion
        prometheus.io/port: {{ .Values.global.gardenlet.metrics.port | quote }}
```